ios/browser/web/web_state/headers.gni

```diff
@@ -0,0 +1,2 @@
+browser_web_webstate_public_headers =
```

Comment:
Can you move this and `brave_web_state.h/mm` to the `api` directory to be consistent with the rest of the things we export headers for

Reply:
Done